python/oneflow/nn/graph/graph.py

```diff
@@ -1197,6 +1197,14 @@ def get_tensor_in_tuple(map_item):
                                 state_tensor_from_eager, self._state_tensor_tuple[s_idx]
                             )
                         self._state_tensor_tuple[s_idx] = state_tensor_from_eager
+                if not with_eager:
+                    for s_idx, s_name in enumerate(self._state_op_names):
+                        if (oneflow.numel(self._state_tensor_tuple[s_idx]) == 0) and (
+                            s_name not in states_from_eager
+                        ):
+                            warnings.warn(
+                                f"Current graph is missing parameter {s_name}, but load_runtime_state_dict needs it. This may cause error later."
```

Comment:
```
nn/graph/graph.py:1205: UserWarning: Current graph is missing parameter my_linear.linear.bias, but load_runtime_state_dict needs it. This may cause er│·························
ror later.                                                                                                                                                                                      │·························
  warnings.warn(                                                                                                                                                                                │·························
F20230324 16:49:02.380128 4159642 register_manager.cpp:146] Check failed: var_blob->AlignedByteSizeOfBlobBody() >= mem_block.mem_size() (0 vs. 512)                                             │·························
*** Check failure stack trace: ***                                                                                                                                                              │·························
    @     0x7f834453063c  google::LogMessage::Fail()                                                                                                                                            │·························
    @     0x7f8344530577  google::LogMessage::SendToLog()                                                                                                                                       │·························
    @     0x7f834452fe62  google::LogMessage::Flush()                                                                                                                                           │·························
    @     0x7f83445339be  google::LogMessageFatal::~LogMessageFatal()                                                                                                                           │·························
    @     0x7f827b6642b8  oneflow::RegstMgr::AddPlan()                                                                                                                                          │·························
    @     0x7f827a6f5141  oneflow::Runtime::Runtime()                                                                                                                                           │·························
    @     0x7f82793ba4e2  oneflow::NNGraph::InitRuntime()   
```

Reply:
如果  miss 的话，这个 tensor 的内存初始化时怎么做呢？

Reply:
> 如果 miss 的话，这个 tensor 的内存初始化时怎么做呢？

没有初始化，后面会报：
```
F20230323 22:27:40.514253 3519595 register_manager.cpp:146] Check failed: var_blob->AlignedByteSizeOfBlobBody() >= mem_block.mem_size() (0 vs. 1024)

    @     0x7ffeebf22c61  oneflow::RegstMgr::AddPlan()
    @     0x7ffeeb61081a  oneflow::Runtime::Runtime()
    @     0x7ffeeaa50033  oneflow::NNGraph::InitRuntime()
```

因为 size 0 可能是对的，所以这里没有 check 而是 warning

Reply:
没看到 Graph C++ 内部对这里的处理

Reply:
嗯，没处理，相当于等着后面 register_manager 报错了。

其它地方没有做 register 到 mem block 的检查